clib/caml/address_class.h

```diff
@@ -0,0 +1,126 @@
+/**************************************************************************/
```

Comment:
Am I correct that all the `caml/` files are copied directly from another project? Are there any changes that we need to review?

Reply:
`caml/` files are copied directly. No need to review.